plugins/pipelines/tokens/create.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const boom = require('boom');
+const schema = require('screwdriver-data-schema');
+const urlLib = require('url');
+
+module.exports = () => ({
+    method: 'POST',
+    path: '/pipelines/{id}/tokens',
+    config: {
+        description: 'Create a new token for pipeline',
+        notes: 'Create a specific token for pipeline',
+        tags: ['api', 'tokens'],
+        auth: {
+            strategies: ['token'],
+            scope: ['user', '!guest']
```

Comment:
Wondering if we should allow pipeline tokens to create/update/delete/etc other pipeline tokens, but I think we can revisit that issue later. This looks fine for now.

Reply:
I think it's better for admins of the pipeline to have control over the management of the pipeline tokens

Reply:
Well right now user tokens can generate more user tokens, so allowing pipeline tokens to create pipeline tokens would be more consistent with what we have.

Reply:
That is logged in users accessing API via UI, If we are not differentiating between generated user access tokens and logged in users then it's a design bug.